protocol/messenger_contacts.go

```diff
@@ -1291,3 +1304,23 @@ func (m *Messenger) forgetContactInfoRequest(publicKey string) {
 
 	delete(m.requestedContacts, publicKey)
 }
+
+func (m *Messenger) GetDeleteForMeMessages() ([]*protobuf.SyncDeleteForMeMessage, error) {
```

Comment:
This isn't really related to contacts

Reply:
lol indeed, I don't know how it got here 😄